src/main/java/com/prgrms/kream/common/mapper/StyleMapper.java

```diff
@@ -1,19 +1,34 @@
 package com.prgrms.kream.common.mapper;
 
-import com.prgrms.kream.domain.member.model.Member;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.prgrms.kream.domain.style.dto.request.GetFeedFacadeRequest;
+import com.prgrms.kream.domain.style.dto.request.GetFeedRequest;
+import com.prgrms.kream.domain.style.dto.request.GetFeedServiceRequest;
+import com.prgrms.kream.domain.style.dto.request.LikeFeedFacadeRequest;
+import com.prgrms.kream.domain.style.dto.request.LikeFeedRequest;
+import com.prgrms.kream.domain.style.dto.request.LikeFeedServiceRequest;
 import com.prgrms.kream.domain.style.dto.request.RegisterFeedFacadeRequest;
 import com.prgrms.kream.domain.style.dto.request.RegisterFeedRequest;
 import com.prgrms.kream.domain.style.dto.request.RegisterFeedServiceRequest;
 import com.prgrms.kream.domain.style.dto.request.UpdateFeedFacadeRequest;
 import com.prgrms.kream.domain.style.dto.request.UpdateFeedRequest;
 import com.prgrms.kream.domain.style.dto.request.UpdateFeedServiceRequest;
+import com.prgrms.kream.domain.style.dto.response.GetFeedFacadeResponse;
+import com.prgrms.kream.domain.style.dto.response.GetFeedFacadeResponses;
+import com.prgrms.kream.domain.style.dto.response.GetFeedResponse;
+import com.prgrms.kream.domain.style.dto.response.GetFeedResponses;
+import com.prgrms.kream.domain.style.dto.response.GetFeedServiceResponse;
+import com.prgrms.kream.domain.style.dto.response.GetFeedServiceResponses;
 import com.prgrms.kream.domain.style.dto.response.RegisterFeedFacadeResponse;
 import com.prgrms.kream.domain.style.dto.response.RegisterFeedResponse;
 import com.prgrms.kream.domain.style.dto.response.RegisterFeedServiceResponse;
 import com.prgrms.kream.domain.style.dto.response.UpdateFeedFacadeResponse;
 import com.prgrms.kream.domain.style.dto.response.UpdateFeedResponse;
 import com.prgrms.kream.domain.style.dto.response.UpdateFeedServiceResponse;
 import com.prgrms.kream.domain.style.model.Feed;
+import com.prgrms.kream.domain.style.model.FeedLike;
 
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
```

Comment:
```suggestion
import lombok.NoArgsConstructor(access = AccessLevel.PRIVATE);
```
기본 생성자를 private 하게 가져가면 좋을 것같아요.